### PR TITLE
ch(travis): Integrate Travis CI to work with Github

### PR DESCRIPTION
**Pull request by XROLE**
#### What does this pull request do?
- This connects Authors-Haven with Travis CI and instructs Travis to run all test written.
#### Description of task to be completed?
- [x] connect Travis CI with Github
- [x] add .travis.yml file
- [x] configure  .travis.yml file to run node js
- [x] configure .travis.yml file to run test
- On successful creating a pull request, you will see a spinning yellow circle on pippin-ah-backend GitHub page telling you that Travis is running some background check
 #### What are the relevant pivotal tracker stories?
- [Finishes #162727540]

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+language: node_js
+node_js:
+ - "10.14.1"
+before_script:
+  - npm install
+script: 
+  - npm run test
```
Comment (oreoluwade) [inline .travis.yml]: Travis should still work as expected even if there's no script directive in this `.travis.yml`, so you could take that out.

By default, it is designed to find and run `npm test`

**Comment by oreoluwade** [comment]
I'd need you to add the travis badge to the `README.md` @XROLE

File: .travis.yml (hunk 1 above)
Comment (XROLE) [reply .travis.yml]: Ok, I never knew about that. Thanks for the feedback. I will be taken it out asap.

Comment (oreoluwade) [reply .travis.yml]: Oh, that's fine @XROLE I know you wouldn't have put it there if you knew.

Comment (XROLE) [reply .travis.yml]: #### Please, I need clarification on this;
- The build passed on Travis CI successfully but the badge has not changed color to show that the test is passing. Do I still add the badge that way or do I wait untill the badge color changes to success (green)?

Comment (XROLE) [reply .travis.yml]: > I'd need you to add the travis badge to the `README.md` @XROLE

Ok. I am adding it right away.

**Comment by XROLE** [comment]
> I'd need you to add the travis badge to the `README.md` @XROLE

Readme badge added successfully

File: README.md [hunk 2]
```diff
@@ -1,4 +1,5 @@
 [![Reviewed by Hound](https://img.shields.io/badge/Reviewed_by-Hound-8E64B0.svg)](https://houndci.com)
+[![Build Status](https://travis-ci.org/andela/pippin-ah-backend.svg?branch=develop)](https://travis-ci.org/andela/pippin-ah-backend)
```
Comment (oreoluwade) [inline README.md]: Please, swap the position of both badges, then when you do that for coveralls as well, put it before the hound badge.

File: .travis.yml (hunk 1 above)
Comment (oreoluwade) [reply .travis.yml]: > #### Please, I need clarification on this;
> * The build passed on Travis CI successfully but the badge has not changed color to show that the test is passing. Do I still add the badge that way or do I wait until the badge color changes to success (green)?

The reason the badge is not changing color is that we need the develop version of the CI. However, at the moment, travis is not on develop yet, so you'd most likely only see an `unknown` status. As long as the travis build passes, when we merge to develop, the badge will be green.

Comment (XROLE) [reply .travis.yml]: Ok. Thanks so much for that.

File: README.md (hunk 2 above)
Comment (XROLE) [reply README.md]: Ok. I'll do it right away

**Review by oreoluwade — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (XROLE) [reply README.md]: Travis CI badge position swapped successfully.

**Pull request merged**